File: Copa.Domain/Chave.cs
using System;

namespace Copa.Domain
{
    public class Chave
    {
        public int Id { get; set; }
        public int Selecao1Id { get; set; }
        public int Selecao2Id { get; set; }
        public DateTime DataConfronto { get; set; }
        public int? QtdGols1 { get; set; }
        public int? QtdGols2 { get; set; }
    }
}

File: Copa-App/src/app/_models/Chave.ts
import { Selecao } from './Selecao';

export interface Chave {
    id: number;
    selecao1Id: number;
    selecao2Id: number;
    dataConfronto: Date;
    qtdGols1?: number;
    qtdGols2?: number;
}
